inc/formanswer.class.php

```diff
@@ -1079,8 +1079,12 @@ public function generateTarget() {
             $success = false;
             break;
          }
-         // Map [itemtype of the target] [item ID of the target] = ID of the generated target
-         $generatedTargets->addTarget($targetObject, $generatedTarget);
+         if ($generatedTarget === true) {
```

Comment:
style : empty if() is not allowed by code style rules. Reverse the condition to solve this issue.